configure.py

```diff
@@ -778,8 +778,11 @@ def find_ninja():
 arg_parser.add_argument('--date-stamp', dest='date_stamp', type=str,
                         help='Set datestamp for SCYLLA-VERSION-GEN')
 arg_parser.add_argument('--use-cmake', action='store_true', help='Use CMake as the build system')
+arg_parser.add_argument('--coverage', action='store_true', help = 'Compile scylla with coverage instrumentation')
```

Comment:
We have --mode=coverage, no?

Reply:
But you're adding three new coverage modes, with an extra flag.

Why not fix the coverage mode?